src/components/error-boundary/Fallback.tsx

```diff
@@ -25,10 +25,20 @@ const Message = styled(View)({
   textAlign: 'center',
 });
 
-export default function Fallback() {
-  const { colors } = useTheme();
+export default function Fallback({
+  colors,
+  error,
+  componentStack,
+  resetError,
+}: {
+  colors: Colors;
+  error: Error;
+  componentStack: string;
+  resetError: () => void;
+}) {
   const handleRestart = () => {
-    logger.sentry('Restarting app after Error Boundary catch');
+    logger.error(new RainbowError('RainbowAppRestartFromErrorBoundary'), { data: { error: error.toString(), componentStack } });
+    resetError();
```

Comment:
this method is recommended from Sentry to call, but unsure if it's needed/wanted. See more here: https://docs.sentry.io/platforms/javascript/guides/react/features/error-boundary/#setting-a-fallback-function-render-props

Reply:
is the expected behavior of this right now that on `resetError` it would just reset the entire `App` (child of the error boundary) and then we would just exit the app on line 42 (`exitApp`) ?

Reply:
not sure what you mean by the "completely cannot get around it" part

Reply:
I'm more wondering if we want a "nuke" option to just reset the app completely if a user finds themselves in an unrecoverable state, but maybe this is already what the above chunk of code does?